script.js

```diff
@@ -0,0 +1,71 @@
+function deliverablesOne(json) {
```

Comment:
interesting approach....  here;s a further for you...

are you able to abstract it into one function and call it deliverable(product, 1);  deliverable(product,2) etc etc...

Reply:
Awesome!!